src/bin/migrate.rs

```diff
@@ -0,0 +1,127 @@
+//! This binary is used to migrate user IDs from base64 to hex encoding.
```

Comment:
Can we do this in a separate PR so we get the base functionality ready for a test run quicker?